File: Assignments/Assignment_17/p9.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                int num=i;
                for(int j=1;j<=i;j++){

                        num*=j;
                        printf("%d\t",num);
                }printf("\n\n");
        }
}

File: Flutter/Assignment/Square pattern set-2/p9.dart
import "dart:io";
void main(){
	print("Enter the no. of rows:");
	int? row=int.parse(stdin.readLineSync()!);
	for(int i=row;i>=1;i--){
		int x=i;
		for(int j=1;j<=i;j++){
			stdout.write("${x}\t");
			x=(x+row-j);
		}
		print("");
	}
}
